packages/runtime-common/text-suggestion.ts

```diff
@@ -0,0 +1,47 @@
+import { isCardTypeFilter, isEveryFilter, type Filter } from './query';
+
+interface ChooseCardSuggestion {
+  suggestion: string; // suggests a UI text
+  depth: number;
+}
+export function suggestCardChooserTitle(
```

Comment:
This function tries to make suggestions as to what text when choosing a list of cards based upon the `Filter`. Particularly, by searching for possible cases of card context like `on` or `type`. 
This function uses a recursive call when going into quantifier like `every`. But not all filters types are supported like `any` or `not` or `range`. The reason is bcos these filters dont contribute to context of the card search and possibly confound